appsec/tests/integration/src/test/groovy/com/datadog/appsec/php/integration/CommonTests.groovy

```diff
@@ -139,6 +139,19 @@ trait CommonTests {
         assert span.meta."_dd.appsec.events.users.login.failure.auto.mode" == 'identification'
     }
 
+    @Test
+    void 'authenticated user event automated'() {
+        def trace = container.traceFromRequest('/behind_auth.php') { HttpResponse<InputStream> resp ->
+            assert resp.statusCode() == 200
+        }
+
+        Span span = trace.first()
+        assert span.metrics._sampling_priority_v1 == 2.0d
+        assert span.meta."usr.id" == 'userID'
```

Comment:
This is a total nitpick, but I rather be able to see a test and understand what's going on just reading the test. On this case, you need to wonder why the id is `userID` since it's not set on the test(it's not the end of the world, just open `behind_auth.php` and done). I would rather be able to pass the id on the request. Something like `/behind_auth.php?userId=[ID_HERE]` and then do assertions against that id. That way you understand where that id is coming from